runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -244,4 +251,26 @@ public static void setOptions(OptionSet options)
 	{
 		RuneLite.options = options;
 	}
+
+	/**
+	 * Enables the osx native fullscreen if running on a mac.
+	 *
+	 * @param gui The gui to enable the fullscreen on.
+	 */
+	private static void tryEnableOSXFullscreen(ClientUI gui)
```

Comment:
This method should be moved to ClientUI class in the create method, as that one contains all the Swing-related code.